README.md

```diff
@@ -83,15 +83,22 @@ Create a new folder under `packages` if it is a common library or create folder
 
 Make sure to name the package as `@cumulus/package-name`.
 
-## Publishing to NPM
+## Versioning
+
+We use a global versioning approach, meaning version numbers in cumulus are consistent across all packages and tasks. We use Lerna to manage our versioning. Any change, whether patch, minor, or major, will force lerna to increment the version of all packages.
```

Comment:
I would mention the versioning scheme (i.e. semantic versioning) and what the version numbers will look like.